src/pages/MyEarnings/MyEarningStats/index.tsx

```diff
@@ -79,7 +80,10 @@ const MyEarningStats = () => {
   const tokensByChainId = useAppSelector(state => state.lists.mapWhitelistTokens)
 
   const elasticEarningQueryResponse = useGetElasticEarningQuery({ account, chainIds: selectedChainIds })
-  const elasticLegacyEarningQueryResponse = useGetElasticLegacyEarningQuery({ account, chainIds: selectedChainIds })
+  const elasticLegacyEarningQueryResponse = useGetElasticLegacyEarningQuery({
+    account,
+    chainIds: selectedChainIds.filter(item => !COMING_SOON_NETWORKS_FOR_MY_EARNINGS_LEGACY.includes(item)),
+  })
```

Comment:
Do we need to memo the param? Can RTK handle this?

Reply:
it can